SPECS-SIGNED/kernel-mshv-signed/kernel-mshv-signed.spec

```diff
@@ -0,0 +1,153 @@
+%global debug_package %{nil}
```

Comment:
Please update the entangled spec checker script for hvloader and kernel-mshv - toolkit/scripts/check_entangled_specs.py